test/correctness/gpu_data_flows.cpp

```diff
@@ -6,10 +6,6 @@ using namespace Halide;
 int main(int argc, char **argv) {
 
     Target target = get_jit_target_from_environment();
-    if (!target.has_gpu_feature() && !target.has_feature(Target::OpenGLCompute)) {
```

Comment:
Seems weird to run these tests if they're not going on a GPU. I assume it's because hexagon scheduling for them is coming?